src/actions/postEventLinkAction.js

```diff
@@ -0,0 +1,20 @@
+import axios from '../helpers/http'
+
+export let postEventLink = props => dispatch => {
+  return axios({
+    method: 'put',
+    url: `/hangouts/${props.id}`,
+    data: {
+      title: props.title,
+      hangout_url: props.link,
+      notify_hangout: true,
+      project_id: props.projectId,
+      event_id: props.id
+      // token: props.cookies.get(process.env.SESSION || 'WebsiteOne_session')
```

Comment:
do we need this line?

Reply:
well, actually this brings up a good point. 
At the moment, I have not tested that this works with a user signed in through GitHub OAuth.

For any actions where a user needs to be authenticated and they are signed in through OAuth, we are passing the `JWT` token in the params, instead of the headers, but I'm sure there is a better way to access the headers that are sent to the backend to decode them, just haven't figured that out yet.... I think it's a bug related to the OAuth, that can be dealt with in a separate ticket. So the above line can be removed.